human.pedestrian.adult:
    default:
        omegaz_mean: 0.0
        omegaz_var: 0.0
        vx_mean: -7.815970093361102e-14
        vx_var: 1013.7979757228796
        vy_mean: 6.927791673660977e-14
        vy_var: 244.49062167783543
        vz_mean: 0.09991367169708487
        vz_var: 0.39659023808666183
    pedestrian.moving:
        omegaz_mean: -0.017503408396545996
        omegaz_var: 1.5976938206521543
        vx_mean: 1.1893421860118478
        vx_var: 0.3493845548540412
        vy_mean: 0.006007075112217132
        vy_var: 0.047404759637401456
        vz_mean: 0.026707740186293773
        vz_var: 0.235504100997478
    pedestrian.standing:
        omegaz_mean: 0.02160706827289617
        omegaz_var: 1.1576402158132335
        vx_mean: 0.0009088345615109643
        vx_var: 0.029834153286953113
        vy_mean: 0.000793518443854007
        vy_var: 0.03001903564741986
        vz_mean: 0.016808625790556263
        vz_var: 0.0862426519520566
    type: dynamic
human.pedestrian.construction_worker:
    pedestrian.moving:
        omegaz_mean: 0.0205612431377648
        omegaz_var: 0.01988717751851533
        vx_mean: -0.05687434843774567
        vx_var: 0.06435509436572097
        vy_mean: 0.08622330872649567
        vy_var: 0.2627549020355487
        vz_mean: -0.006744245846483046
        vz_var: 0.09277354381758977
    pedestrian.standing:
        omegaz_mean: -0.0002695328284741749
        omegaz_var: 0.007523667024018812
        vx_mean: 0.014727208118669944
        vx_var: 0.05890790143176429
        vy_mean: 0.014262265880330197
        vy_var: 0.062032971379281474
        vz_mean: 0.013867753810819767
        vz_var: 0.024856619372965502
    type: dynamic
movable_object.barrier:
    default:
        px_var: 0.0009171156563256157
        py_var: 0.0031720214989580608
        pz_var: 0.041303231199257204
    type: static
movable_object.debris:
    default:
        px_var: 0.009647928994082615
        py_var: 0.0016951005917163488
        pz_var: 0.002187562130177511
    type: static
movable_object.pushable_pullable:
    default:
        px_var: 0.0003556528925621124
        py_var: 0.0011348780991749486
        pz_var: 0.035857082644628085
    type: static
movable_object.trafficcone:
    default:
        px_var: 0.0022707322759421456
        py_var: 0.002500784499213021
        pz_var: 0.02044077933713115
    type: static
vehicle.bicycle:
    cycle.with_rider:
        omegaz_mean: 0.0
        omegaz_var: 0.0
        vx_mean: 0.008566471665576914
        vx_var: 0.07573544094390101
        vy_mean: 0.020897078770001905
        vy_var: 0.20738495738870988
        vz_mean: 0.05304967060870632
        vz_var: 0.1381780775588083
    cycle.without_rider:
        omegaz_mean: -0.007087751552347337
        omegaz_var: 0.10480255492153334
        vx_mean: 0.005008545832545416
        vx_var: 0.0035366796898119376
        vy_mean: -0.0005571451692489635
        vy_var: 9.67432850513216e-06
        vz_mean: 0.0037694607353915338
        vz_var: 0.04182068728686706
    type: dynamic
vehicle.bus.rigid:
    type: dynamic
    vehicle.stopped:
        omegaz_mean: 0.0
        omegaz_var: 0.0
        vx_mean: 0.0
        vx_var: 0.0
        vy_mean: 0.0
        vy_var: 0.0
        vz_mean: 0.07312803940822035
        vz_var: 0.8138356889463483
vehicle.car:
    type: dynamic
    vehicle.moving:
        omegaz_mean: -0.023187047786386496
        omegaz_var: 2.7882903495108304
        vx_mean: 4.846730529702576
        vx_var: 6.78515258912643
        vy_mean: 0.03952407269396666
        vy_var: 0.265745050370708
        vz_mean: 0.0341896769056306
        vz_var: 0.08475458343273112
    vehicle.stopped:
        omegaz_mean: -5.7858325313829735e-05
        omegaz_var: 8.801437412523545e-07
        vx_mean: 0.004945169044334655
        vx_var: 0.037979026054243754
        vy_mean: 0.00899381666305805
        vy_var: 0.0519921189367568
        vz_mean: 0.027918532178103878
        vz_var: 0.16057950977381963
vehicle.construction:
    type: dynamic
    vehicle.stopped:
        omegaz_mean: 0.0
        omegaz_var: 0.0
        vx_mean: 0.0
        vx_var: 0.0
        vy_mean: 0.0
        vy_var: 0.0
        vz_mean: 0.017932720706674427
        vz_var: 0.021133245418134362
vehicle.motorcycle:
    cycle.with_rider:
        omegaz_mean: 0.0
        omegaz_var: 0.0
        vx_mean: 0.02788339015354424
        vx_var: 1.3387021217766646
        vy_mean: 0.0045338267147580135
        vy_var: 0.05284945443223044
        vz_mean: 0.039308513206265376
        vz_var: 0.8402762071172539
    cycle.without_rider:
        omegaz_mean: 0.0
        omegaz_var: 0.0
        vx_mean: -6.485722961283056e-05
        vx_var: 3.491291879394781e-06
        vy_mean: -0.007027078829882938
        vy_var: 0.021010490502044617
        vz_mean: 0.007845407939476669
        vz_var: 0.005744269313804246
    type: dynamic
vehicle.truck:
    type: dynamic
    vehicle.moving:
        omegaz_mean: -0.041125423532598224
        omegaz_var: 0.006497441521308371
        vx_mean: 4.892754536420509
        vx_var: 10.955038182808547
        vy_mean: -0.0409925443323776
        vy_var: 0.09733913540231265
        vz_mean: 0.09051725339036297
        vz_var: 0.1919684101093357
    vehicle.parked:
        omegaz_mean: -0.0010178129391307533
        omegaz_var: 0.0005272950781425567
        vx_mean: -0.008694525830724243
        vx_var: 0.266276857506972
        vy_mean: 0.039158641866313966
        vy_var: 0.337982628995192
        vz_mean: 0.008756763857386602
        vz_var: 0.32914451803902245
    vehicle.stopped:
        omegaz_mean: 0.0
        omegaz_var: 0.0
        vx_mean: 0.0018385747567303994
        vx_var: 0.0011673578064246319
        vy_mean: -0.002803754835955374
        vy_var: 0.00015918197746849028
        vz_mean: 0.013883324576363463
        vz_var: 0.3065456798239543
